src/main/java/org/ideoholic/curium/model/periods/action/PeriodAction.java

```diff
@@ -38,6 +38,10 @@ public class PeriodAction {
 	@Autowired
 	HttpSession httpSession;
 	@Autowired
+	private DocumentService DocumentService;
```

Comment:
Remove this variable `private DocumentService DocumentService;` here.
It is not needed in this class.